app/views/recipes/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <div class="col-xs-5 col-xs-offset-1"> %>
     <div class="well well-lg" id:"test">
       <div class="text-center">
-        <strong><%= link_to recipe["title"], recipes_show_path(param :id) %></strong>
+        <strong><%= link_to recipe["title"], recipes_show_path(recipe.id) %></strong>
```

Comment:
for the argument to the `recipes_show_path` helper I used a named parameter. i.e. `recipes_show_path(id: recipe.id)`